### PR TITLE
Support mod_shib auth

### PR DESCRIPTION
**Pull request by jsturek**
You can test shibboleth user auth on https://mediahub-test.unl.edu/.

### CODE REVIEW

File: www/templates/html/RuntimeException.tpl.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      wdn.initializePlugin('notice');
+    });
+  });
+</script>
+<div class="dcf-mt-8 dcf-mb-8 wdn_notice alert" style="padding: 6px;">
+  <div class="message" style="color: #fff;">
+    <h1 class="title"><?php echo $title; ?></h1>
+    <p><?php echo UNL_MediaHub::escape($context->getMessage()); ?></p>
+  </div>
+</div>
```
Comment (skoolbus39) [inline www/templates/html/RuntimeException.tpl.php:39]: No newline at end of file

File: www/templates/html/DefaultHomepage.tpl.php [hunk 2]
```diff
@@ -39,8 +39,8 @@
     <div class="mh-featured">
       <a class="dcf-d-flex dcf-flex-col dcf-ai-center dcf-txt-decor-hover" href="<?php echo $baseUrl ?>search/">
         <div class="mh-featured-icon mh-green dcf-d-flex dcf-ai-center dcf-jc-center dcf-h-10 dcf-w-10 dcf-circle">
-          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>/templates/html/css/images/play-icon.svg"  title="Play Icon">
-            <img src="<?php echo $baseUrl; ?>/templates/html/css/images/play-icon-white.png" alt="browse media">
+          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>templates/html/css/images/play-icon.svg"  title="Play Icon">
+            <img src="<?php echo $baseUrl; ?>templates/html/css/images/play-icon-white.png" alt="browse media">
```
Comment (skoolbus39) [inline www/templates/html/DefaultHomepage.tpl.php:43]: The SVG icons appear as white squares now. I assume it has to do with the updated file paths?

**Review by skoolbus39 — CHANGES_REQUESTED** [review]

File: www/templates/html/DefaultHomepage.tpl.php [hunk 3]
```diff
@@ -50,8 +50,8 @@
     <div class="mh-featured">
       <a class="dcf-d-flex dcf-flex-col dcf-ai-center dcf-txt-decor-hover" href="<?php echo $baseUrl ?>channels/">
         <div class="mh-featured-icon mh-blue dcf-d-flex dcf-ai-center dcf-jc-center dcf-h-10 dcf-w-10 dcf-circle">
-          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>/templates/html/css/images/channel-icon.svg" title="Default Channel Icon">
-            <img src="<?php echo $baseUrl; ?>/templates/html/css/images/channel-icon-white.png" alt="explore channels">
+          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>templates/html/css/images/channel-icon.svg" title="Default Channel Icon">
+            <img src="<?php echo $baseUrl; ?>templates/html/css/images/channel-icon-white.png" alt="explore channels">
```
Comment (skoolbus39) [inline www/templates/html/DefaultHomepage.tpl.php:54]: The SVG icons appear as white squares now. I assume it has to do with the updated file paths?

File: www/templates/html/DefaultHomepage.tpl.php [hunk 4]
```diff
@@ -61,8 +61,8 @@
     <div class="mh-featured">
       <a class="dcf-d-flex dcf-flex-col dcf-ai-center dcf-txt-decor-hover" href="<?php echo $baseUrl ?>manager/">
         <div class="mh-featured-icon dcf-d-flex dcf-ai-center dcf-jc-center dcf-h-10 dcf-w-10 dcf-circle unl-bg-scarlet">
-          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>/templates/html/css/images/gear-icon.svg"  title="Manage Media Icon">
-            <img src="<?php echo $baseUrl; ?>/templates/html/css/images/gear-icon-white.png" alt="manage media">
+          <object type="image/svg+xml" data="<?php echo $baseUrl; ?>templates/html/css/images/gear-icon.svg"  title="Manage Media Icon">
+            <img src="<?php echo $baseUrl; ?>templates/html/css/images/gear-icon-white.png" alt="manage media">
```
Comment (skoolbus39) [inline www/templates/html/DefaultHomepage.tpl.php:65]: The SVG icons appear as white squares now. I assume it has to do with the updated file paths?

Comment (jsturek) [reply www/templates/html/DefaultHomepage.tpl.php:65]: @skoolbus39 - The path updates removed an extra `/` from the file path.  I don't see the issue.  Maybe do a hard refresh.  I rather not have the extra '/'.

**Pull request merged**